### ISSUE TITLE
Unable to use Repos Scanning with .Net Core 2.2 project using Directory.Build.props

### ISSUE THREAD

**Issue by cwoolum** [snyk/snyk#770]
- `node -v`: N/A
- `npm -v`: N/A
- `snyk -v`: N/A
- Command run: N/A - Repo Scanning

This builds off of issue #143 by @jessehouwing. All `*.props`, `*.targets`, and `*.*proj` files can contain metadata for a project.

### Expected behaviour
We use `Directory.Build.targets` to load a `dependencies.props` file that contains all of our package versions in a central location. When the repo is scanned, the scanner should detect `Directory.Build.props` at the root of the repo and follow it to build any necessary metadata.

### Actual behaviour
When the project is scanned, since versions are not specified inline, the scanner resolves their lowest published version instead of the one specified in `dependencies.props`

### Steps to reproduce

The EFCore repo is a great example of this

https://github.com/aspnet/EntityFrameworkCore

---

If applicable, please append the `--debug` flag on your command and include the output here **ensuring to remove any sensitive/personal details or tokens.

**Comment by jessehouwing** [comment]
In .NET core, scanning the project.assets.json after build will always take into account the actually resolved dependencies. See also: 

https://www.gdbc-challenges.com/Challenges/PostMortemSummary/VULNPACKAGE

**Comment by cwoolum** [comment]
Yes, these versions do get scanned correctly when Snyk is used as part of our build process. This bug is directly targeted at repo scanning where project.assets.json doesn't exist.
